packages/connection/tests/php/test_Manager.php

```diff
@@ -7,15 +7,16 @@
 use PHPUnit\Framework\TestCase;
 
 use Automattic\Jetpack\Constants;
+use Automattic\Jetpack\Connection\Utils;
 
 class ManagerTest extends TestCase {
 
 	protected $arguments_stack = [];
 
 	public function setUp() {
 		$this->manager = $this->getMockBuilder( 'Automattic\Jetpack\Connection\Manager' )
-		                      ->setMethods( [ 'get_access_token' ] )
-		                      ->getMock();
+							  ->setMethods( array( 'get_access_token' ) )
+							  ->getMock();
```

Comment:
Sorry about these white space changes in this PR. The pre-commit hook automatically made these fixes.